.env.example

```diff
@@ -7,4 +7,9 @@ EXPOSED_DB_PORT=5432
 POSTGRES_PASSWORD=docker
 POSTGRES_USER=docker
 POSTGRES_DB=teacher_workout
-SEEDDATA=false
\ No newline at end of file
+SEEDDATA=false
+SEEDUSERS=false
+ADMIN_EMAIL=
+ADMIN_PASSWORD=
+
+JWT_SECRET=
```

Comment:
Can we add default values so that someone new to the project can just copy .env.example -> .env and is able to run the project?